### ISSUE TITLE
Broken test on master - has been for a while

### ISSUE THREAD

**Issue by rhisiart-cufflink** [jonataslaw/get_storage#13]
https://github.com/jonataslaw/get_storage/runs/956543996?check_suite_focus=true

```
00:43 +0 -1: loading /Users/runner/work/get_storage/get_storage/test/getstorage_test.dart [E]                                                                                                          
  Failed to load "/Users/runner/work/get_storage/get_storage/test/getstorage_test.dart": MissingPluginException(No implementation found for method getApplicationDocumentsDirectory on channel plugins.flutter.io/path_provider)
  package:get_storage/src/storage_impl.dart 48:7   GetStorage._init
  ===== asynchronous gap ===========================
  package:get_storage/src/storage_impl.dart 28:13  new GetStorage._internal.<fn>
  

00:43 +0 -1: Some tests failed.                                                                                                                                                                        
##[error]Process completed with exit code 1.
```

**Comment by jonataslaw** [comment]
You can do the tests locally, the flutter has a problem in path_provider that is not being recognized in tests. All storage libraries have been broken for at least 2 months.

**Comment by rhisiart-cufflink** [comment]
That's what I'm doing...

```
MacBook:get_storage rhisiart$ flutter test test/getstorage_test.dart 
00:02 +0 -1: loading /Users/rhisiart/Cufflink.io/git/get_storage/test/getstorage_test.dart [E]                                                    
  Failed to load "/Users/rhisiart/Cufflink.io/git/get_storage/test/getstorage_test.dart": MissingPluginException(No implementation found for method getApplicationDocumentsDirectory on channel plugins.flutter.io/path_provider)
  package:get_storage/src/storage_impl.dart 48:7   GetStorage._init
  ===== asynchronous gap ===========================
  package:get_storage/src/storage_impl.dart 28:13  new GetStorage._internal.<fn>
  ```

**Comment by jonataslaw** [comment]
Try adding and then immediately removing the pubspec path_provider plugin, it is an alternative work.
And that doesn't depend on us, but on the Flutter team that maintains the path_provider

**Comment by rhisiart-cufflink** [comment]
Fixed in #18
